### PR TITLE
fix: mq support bundle flag change follow up

### PR DESCRIPTION
**Pull request by Elsie4ever**
---
This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

Thank you for contributing to Azure IoT Operations tooling!

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

Intent for Production

- [ ] It is expected that pull requests made to default or core branches such as `dev` or `main` are of production grade. Corollary to this, any merged contributions to these branches may be deployed in a public release at any given time. By checking this box, you agree and commit to the expected production quality of code.

Basic expectations

- [ ] If introducing new functionality or modified behavior, are they backed by unit and/or integration tests?
- [ ] In the same context as above are command names and their parameter definitions accurate? Do help docs have sufficient content?
- [ ] Have **all** the relevant unit **and** integration tests pass? i.e. `pytest <project root> -vv`. Please provide evidence in the form of a screenshot showing a succesful run of tests locally OR a link to a test pipeline that has been run against the change-set.
- [ ] Have linter checks passed using the `.pylintrc` and `.flake8` rules? Look at the CI scripts for example usage.
- [ ] Have extraneous print or debug statements, commented out code-blocks or code-statements (if any) been removed from the surface area of changes?
- [ ] Have you made an entry in HISTORY.rst which concisely explains your user-facing feature or change?

Azure IoT Operations CLI maintainers reserve the right to enforce any of the outlined expectations.

A PR is considered **ready for review** when all basic expectations have been met (or do not apply).

### CODE REVIEW

File: azext_edge/tests/edge/support/create_bundle_int/test_auto_int.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def test_create_bundle(init_setup, bundle_dir, mq_traces, ops_service, tracked_f
     if ops_service == OpsServiceType.dataprocessor.value and not DATA_PROCESSOR_API_V1.is_deployed():
         pytest.skip("Data processor is not deployed on this cluster.")
 
-    command = f"az iot ops support create-bundle --mq-traces {mq_traces} " + "--ops-service {0}"
+    command = f"az iot ops support create-bundle --broker-traces {mq_traces} " + "--ops-service {0}"
```
Comment (vilit1) [inline azext_edge/tests/edge/support/create_bundle_int/test_auto_int.py:40]: im going to cry when there is broker as a service and broker as an object and broker as everything

Comment (Elsie4ever) [reply azext_edge/tests/edge/support/create_bundle_int/test_auto_int.py:40]: what do you mean?😣

**Review by vilit1 — APPROVED** [review]
want int test run

File: azext_edge/edge/providers/support/base.py [hunk 2]
```diff
@@ -164,7 +164,7 @@ def process_statefulset(
     return_namespaces: bool = False,
     field_selector: Optional[str] = None,
     label_selector: Optional[str] = None,
-) -> List[dict]:
+) -> Union[Tuple, List[dict]]:
```
Comment (digimaun) [inline azext_edge/edge/providers/support/base.py:167]: when using Tuple from typing, might as well provide types for the generic i.e. `Union[Tuple[List[dict], dict], List[dict]]`

**Review by digimaun — APPROVED** [review]

**Pull request merged**